Homework3/task3_hw3.py

```diff
@@ -0,0 +1,43 @@
+base_url = "https://codefirstgirls.com/"
+current_url = base_url
+history = []
+
+while True:
+    print("You are currently on the URL", current_url)
+
+    if current_url == base_url:
```

Comment:
Instead of using if else conditions you can use a dictionary to store the url and options as key value pairs. This will improve the efficiency of your program.